TargetFinder

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+
+# Author:                Paul Chojecki
+# Version:               1.0
+# pull request date :    10/4/2018
+
+# Description:
+"""
+This program was written for the purpose of detecting square targets. Although the program is intended to detect square targets, it works by detecting any shape having four straight edges. The program displays the detected target by displaying the area, number of sides, and the name of the shape. The overall execution of the program is as follows:
+
+	* The program converts the input image to gray scale	
+	* The program blurs the resulting image
+	* The program applies a threshold on the resulting image
+	* The program applies a Laplace filter on the resulting image
+	* The program dilates any white spaces
+	* The program identifies all contours
+
+For all contrours the program:
+	
+	* calculates the centroid of the shape
+	* calculates the area of the shape
+
+For all contrours having an area above a certain threshold the program:
+
+	* calculates/diplays the shape name/contour
+	* calculates/displays the number of sides
+	* calculates/displays the area
+
+"""
+
+from __future__ import print_function
+
+import roslib
+roslib.load_manifest('beginner_tutorials')
+import sys
+import rospy
+import cv2
+import imutils
```

Comment:
This isn't installed on NaviGator (will crash program). Either remove the dependency or add it to the install script.